README.md

```diff
@@ -89,6 +89,7 @@ Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
 - [ipfs-embed](https://github.com/ipfs-rust/ipfs-embed) - A small embeddable ipfs implementation
 used and maintained by [Actyx][https://www.actyx.com].
 - [iroh](https://github.com/n0-computer/iroh) - Next-generation implementation of IPFS for Cloud & Mobile platforms.
+- [HotShot](https://github.com/EspressoSystems/HotShot) - Decentralized sequencer in Rust developed by [Espresso Systems](https://www.espressosys.com/).
```

Comment:
Thanks for the addition. If I am not missing something, this is in alphabetical order. Mind ordering your entry accordingly?

Reply:
Whoops! Should be ordered alphabetically now.